games-strategy/0ad/0ad-0.27.0.ebuild

```diff
@@ -0,0 +1,235 @@
+# Copyright 2014-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_12 python3_13)
```

Comment:
I would use the shorter:
PYTHON_COMPAT=( python3_{12..13} )

form (it will also be easier to update in the future for new python versions